Remove init command (#569)
aerogear/graphback: docs/gettingstarted.md

sidebar_label: What is Graphback
---

> Graphback is an package andCLI tool that can help you bootstrap a GraphQL server and client for you including GraphQL schema, resolvers and generating database structure. Graphback addresses the difficulty in configuring a GraphQL server from scratch by giving developers:

Reviewer: ```suggestion
Graphback is a package and CLI tool that can help you bootstrap a GraphQL server and client for you including GraphQL schema, resolvers and generating database structure. Graphback addresses the difficulty in configuring a GraphQL server from scratch by giving developers:
```

- Generated GraphQL Schema containing Queries and Mutations for general data access
- Underlying resolvers to fetch data in performant way
- Support for subscriptions and live updates on the data
- Relational support
- Database migrations 

> - Extendable runtime layer for composing data services used in generated resolvers

Reviewer: ```suggestion
- Extensible runtime layer for composing data services used in generated resolvers
```

- Generated client side queries 

## Getting started using GraphQL CLI

You can use any of the available GraphQL-CLI templates to bootstrap Graphback enabled project. 

With yarn:
```
yarn global add graphql-cli@canary
```

With npm:
```
npm install -g graphql-cli@canary
```

Execute init command to create project from scratch

```
graphql init
```

Please answer all questions.
After project is created please follow project README file that might differ depending on

> the template tha was choosen on startup.

Reviewer: ```suggestion
the template that was chosen on startup.
```

> GraphQL CLI offers various commands to interact with your project that work along Graphback package.

Reviewer: ```suggestion
GraphQL CLI offers various commands to interact with your project that work with the Graphback package.
```

Graphback specific commands are available by executing `graphql generate` 
 
## Adding Graphback to existing project using Graphback CLI

```bash
npm i -g graphback-cli

> graphback init <project-name>

Reviewer: Aren't we removing the `init` command?
